clouddq/BUILD

```diff
@@ -56,6 +56,7 @@ py_binary(
         requirement("click"),
         requirement("coloredlogs"),
         requirement("sqlite-utils"),
+        requirement("markupsafe"),
```

Comment:
Just curious, where do we actually use `markupsafe` lib from the code?

Reply:
we use jinja, jinja internally has a dependency on markupsafe.